fn capture accepts 3 arguments but examples have 4.

In the examples and README's the capture function accepts 4 arguments.

```rust
stackdump_capture::cortex_m::capture(
                &mut *STACK_CAPTURE.as_mut_ptr(),
                &mut *CORE_REGISTERS_CAPTURE.as_mut_ptr(),
                &mut *FPU_REGISTERS_CAPTURE.as_mut_ptr(),
                &cs,
            );
```
But it is defined to accept 3 arguments in the source. 
```rust
/// Capture the core & fpu registers and the stack
#[cfg(has_fpu)]
pub fn capture<const SIZE: usize>(
    stack: &mut ArrayMemoryRegion<SIZE>,
    core_registers: &mut ArrayRegisterData<16, u32>,
    fpu_registers: &mut ArrayRegisterData<32, u32>,
) {
    capture_core_registers(core_registers);
    capture_fpu_registers(fpu_registers);
    capture_stack(
        core_registers
            .register(stackdump_core::gimli::Arm::SP)
            .unwrap(),
        stack,
    );
}
```

resulting in this error:
```
error[E0061]: this function takes 3 arguments but 4 arguments were supplied
  --> src/common.rs:53:13
   |
53 |               stackdump_capture::cortex_m::capture(
   |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
...
56 |                   &mut *FPU_REGISTERS_CAPTURE.as_mut_ptr(),
   |  _________________________________________________________-
57 | |                 &cs,
   | |                 ---
   | |_________________|_|
   |                   | help: remove the extra argument
   |                   unexpected argument of type `&&CriticalSection`
   |
note: function defined here
  --> /Users/mali/.cargo/registry/src/index.crates.io-6f17d22bba15001f/stackdump-capture-0.4.0/src/cortex_m.rs:23:8
   |
23 | pub fn capture<const SIZE: usize>(
   |        ^^^^^^^
```

--- a/capture/README.md
+++ b/capture/README.md
@@ -37,9 +37,8 @@ let mut stack_capture = ArrayMemoryRegion::default();
 let mut core_registers = ArrayRegisterData::default();
 let mut fpu_registers = ArrayRegisterData::default();
 
-cortex_m::interrupt::free(|cs| {
-    stackdump_capture::cortex_m::capture(&mut stack_capture, &mut core_registers, &mut fpu_registers, &cs)
-});
+
+stackdump_capture::cortex_m::capture(&mut stack_capture, &mut core_registers, &mut fpu_registers);
 ```
 
 ## For use when crashing (using cortex m as example target)
@@ -102,19 +101,16 @@ Now we can capture a everything in e.g. a panic.
 ```rust,ignore
 #[panic_handler]
 fn panic(_info: &core::panic::PanicInfo) -> ! {
-    cortex_m::interrupt::free(|cs| {
-        unsafe {
-            stackdump_capture::cortex_m::capture(
-                &mut *STACK_CAPTURE.as_mut_ptr(),
-                &mut *CORE_REGISTERS_CAPTURE.as_mut_ptr(),
-                &mut *FPU_REGISTERS_CAPTURE.as_mut_ptr(),
-                &cs
-            );
-            // If you want to capture the heap or the static data, then do that here too yourself
-        }
+    unsafe {
+        stackdump_capture::cortex_m::capture(
+            &mut *STACK_CAPTURE.as_mut_ptr(),
+            &mut *CORE_REGISTERS_CAPTURE.as_mut_ptr(),
+            &mut *FPU_REGISTERS_CAPTURE.as_mut_ptr(),
+        );
+        // If you want to capture the heap or the static data, then do that here too yourself
+    }
 
-        set_capture_made();
-    });
+    set_capture_made();
     cortex_m::peripheral::SCB::sys_reset()
 }
 ```

